users/tuning.rst

```diff
@@ -214,3 +214,35 @@ Other things:
     number concurrently running threads Syncthing uses. Setting it to 1 (or
     any number lower than your actual number of cores) will reduce the
     amount of CPU used by Syncthing at any given moment.
+
+Tuning for LAN-only
+-------------------
+
+You are paranoid about security and/or privacy, and want to keep Syncthing's
+communication constrained to your local network only.
+
+For restricting sync traffic to only LAN:
+
+- :opt:`relaysEnabled`
+    Set to false to disable relaying sync traffic through servers other than
+    your sync partners. When relays are used, those that you connect to can
+    see your encrypted data, device ID, and public IP address.
```

Comment:
This sounds a bit misleading to me.  "See your encrypted data" could give a false impression that the relay can actually see the data *although* it is encrypted.  The truth is, it can only see some gibberish byte stream *because* the data stays encrypted while passing through the relay.

```suggestion
    see your device ID and public IP address, but not the synchronized data,
    as that is encrypted.
```

Feel free to improve the wording, it's just a quick draft.